src/py42/clients/savedsearch.py

```diff
@@ -32,28 +32,28 @@ def get_by_id(self, search_id):
         uri = u"{}/{}".format(self._resource, search_id)
         return self._session.get(uri)
 
-    def get_query(self, search_id):
+    def get_query(self, search_id, pg_num=None):
         """Get the saved search in form of a query(`py42.sdk.queries.fileevents.file_event_query`).
 
         Args:
             search_id (str): Unique search Id of the saved search.
+            pg_num (int, optional): The consecutive group of results of size pg_size in the result set to return. Defaults to 1.
         Returns:
             :class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery`
         """
         response = self.get_by_id(search_id)
         search = response[u"searches"][0]
-        return FileEventQuery.from_dict(search)
+        return FileEventQuery.from_dict(search, page_number=pg_num)
 
-    def execute(self, search_id, pg_num=1, pg_size=10000):
```

Comment:
I think I was not clear in my last advice! I still think that `pg_size` still should be a parameter, but it should just default to None here since somewhere else it automatically turns into the default value on its own.

Reply:
Oh sorry, I removed it in favour of `settings.security_events_per_page` because I thought being able to set it in two places would be potentially confusing, but now I realise it makes sense to add it back because then a user can set `settings.security_events_per_page` but still provide the parameter if needed for individual calls.

Reply:
Exactly... The settings is the default. The user does not even have to know about the setting to use the parameter. I personally kind of expect `pg_num` and `pg_size` as parameters in methods that page, that is why we have `get_all()` that ignores them and `get_page()` for allowing them. Now with these changes, we can have them in saved search as well!